sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -589,3 +589,15 @@ case class AnalyzeColumn(
     "mutually exclusive. Only one of them should be specified.")
   override def children: Seq[LogicalPlan] = child :: Nil
 }
+
+/**
+ * The logical plan of the LOAD DATA INTO TABLE command that works for v2 catalogs.
```

Comment:
We should remove  `that works for v2 catalogs` in all the commands in this file. It works for both v1 and v2 catalogs.

Reply:
I removed all the instances of `that works for v2 catalogs`.

Btw, there are some commands that work only with v1 catalogs - namespace related commands, for example.  (this is a commit removing the comments only for commands that work for both v1/v2 catalogs.)